dotcom-rendering/src/web/server/onwardsToHtml.tsx

```diff
@@ -0,0 +1,41 @@
+import { renderToString } from 'react-dom/server';
+import { Carousel } from '../components/Carousel';
+import { decideFormat } from '../lib/decideFormat';
+import { decideTrail } from '../lib/decideTrail';
+
+const buildTrails = (
+	trails: CAPITrailType[],
+	trailLimit: number,
+): TrailType[] => {
+	return trails.slice(0, trailLimit).map(decideTrail);
+};
+
+/**
+ * onwardsToHtml is used by the /Onwards endpoint to render the Carousel at the bottom of articles
+ * It takes an array of json key-event blocks and returns the resulting html string
+ *
+ * @returns string (the html)
+ */
+export const onwardsToHtml = ({
+	heading,
+	// description,
+	// url,
```

Comment:
These two properties are not currently supported by `Carousel` but were used by the previous onwards components. I've left them here for now until we see what we decide to pass in from Frontend.